### ISSUE TITLE
[BUG] Contradiction in Readme about fetch

### ISSUE THREAD

**Issue by StefanSchoof** [microsoft/ApplicationInsights-JS#2005]
**Description/Screenshot**
The readme stat at one place the fetch collection is disabled by default and on an other place that it is active by default:

[Setting Up Autocollection](https://github.com/microsoft/ApplicationInsights-JS/blob/master/README.md#setting-up-autocollection)

> Network Dependency Requests made by your app XHR and Fetch (fetch collection is disabled by default) requests, include information on 

[Configuration](https://github.com/microsoft/ApplicationInsights-JS/blob/master/README.md#configuration)
> disableFetchTracking | boolean | false | If true, Fetch requests are not autocollected. Default is false (Since v2.8.0, previously true)

I assume, that "fetch collection is disabled by default" has been not updated, when changing the default in 2.8.0

**Steps to Reproduce**

 - OS/Browser:
 - SDK Version [e.g. 22]:
 - How you initialized the SDK:

**Expected behavior**

No Contradiction about fetch in the documentation.

**Additional context**

**Comment by MSNev** [comment]
Correct, we need to update the documentation to indicate that it is now enabled by default (since 2.8.0)

Thank you.
